cloud-controller-manager/do/loadbalancers.go

```diff
@@ -245,6 +256,15 @@ func (l *loadBalancers) GetLoadBalancerName(_ context.Context, clusterName strin
 }
 
 func getDefaultLoadBalancerName(service *v1.Service) string {
```

Comment:
We might want to rename this method, as it is no `Default` name anymore. Also it appears that it is being used in a different places too.